front/pages/w/[wId]/ds/new.tsx

```diff
@@ -347,6 +349,31 @@ export default function DataSourceNew({
                       </p>
                     </div>
                   </div>
+                  <fieldset className="mt-6">
```

Comment:
Can you share an updated screenshot.

I think we don't need the "Upload rights" section thingy

Reply:
![image](https://github.com/dust-tt/dust/assets/52892257/04c3bdfb-8c97-4a5e-995f-c82d3f59bd67)

vs

![image](https://github.com/dust-tt/dust/assets/52892257/bcbec934-1e4f-4e5e-9bac-2efc7356831e)